src/components/TableOfContents.js

```diff
@@ -30,46 +30,14 @@ const TableOfContents = ({ page }) => {
       });
   }, [mdxAST]);
 
-  const raf = useRef();
-  const navRef = useRef();
-  const activeRef = useRef();
   const headingIds = useMemo(() => headings.map(prop('id')), [headings]);
   const activeHash = useActiveHash(headingIds);
-  const previousActiveHash = usePrevious(activeHash);
-  const changedActiveHash = activeHash !== previousActiveHash;
-
   const isMobileScreen = useMedia('(max-width: 1240px)');
 
-  useEffect(() => {
-    if (!activeRef.current) {
-      return;
-    }
-
-    const navRect = navRef.current.getBoundingClientRect();
```

Comment:
How is it scrolling now? Where does that behavior come from?

Reply:
So this scroll behavior you see deleted was actually controlling the scroll of the table of contents. This was more of a nice feature than not. Basically this just analyzed if the current "active hash" changed, and whether or not it was visible in the right rail. If it wasn't visible, it would scroll the right rail to show that link.

Unfortunately it seems like this scroll behavior caused it to capture the event awkwardly or something, which prevented the page itself from scrolling. Not entirely sure why, but removing this functionality seemed to fix it. 

Since this was more of a nice feature than a must have, I went ahead and removed it. Its a tricky problem to solve correctly. For now, this just means the user would have to scroll the right rail if they scroll to a header that is not visible.

Reply:
Fair enough! thank you for the explanation